src/python/grpcio/grpc/_utilities.py

```diff
@@ -189,3 +189,32 @@ def channel_ready_future(channel: grpc.Channel) -> _ChannelReadyFuture:
     ready_future = _ChannelReadyFuture(channel)
     ready_future.start()
     return ready_future
+
+
+def first_version_is_lower(version1: str, version2: str) -> bool:
+    """
```

Comment:
Please add some simple unit tests for this function. I used the following just to poke at it, but you may want to add some more:

```
print(first_version_is_lower("1.2.3", "1.2.4"))
print(first_version_is_lower("1.2.4", "1.2.3"))
print(first_version_is_lower("1.2.3dev0", "1.2.3"))
print(first_version_is_lower("1.2.3", "1.2.3dev0"))
```